regparser/web/jobs/views.py

```diff
@@ -58,23 +76,24 @@ def create(self, request, *args, **kwargs):
         """
         serialized = self.get_serializer(data=request.data)
         serialized.is_valid(raise_exception=True)
-        eregs_args = [
-            "pipeline",
-            str(serialized.validated_data["cfr_title"]),
-            str(serialized.validated_data["cfr_part"]),
-            "./testing"
-        ]
+
+        eregs_args = self.build_eregs_args(serialized.validated_data)
         job = queue_eregs_job(eregs_args, timeout=60*30, result_ttl=-1)
 
         # Paranoia--validate the values we provide:
         job_id = job.id
         for validator in serialized.get_fields()["job_id"].validators:
             validator(job_id)
-        statusurl = status_url(job_id)
+        statusurl = status_url(job_id, sub_path=self.sub_path)
```

Comment:
Good point. I'm not sure that my approach here is correct; it feels wrong to have to tell the view what part of its URL is (particularly since this will require changing strings in multiple places if we, for example, renamed "proposal" to "notice"). I'm not sure what the correct canonical location for this information should be, though.